### ISSUE TITLE
Errors when loading .rs files after commit eb716cb

### ISSUE THREAD

**Issue by iirelu** [rust-lang/rust.vim#120]
Sample error:

```
"src/libsyntax/parse/mod.rs" 1097L, 43180C
Error detected while processing function <SNR>13_LoadFTPlugin:
line    2:
E216: No such group or event: rust.vim|augroup! rust.vim
Press ENTER or type command to continue
```

This happens on every load of a Rust file. No errors before eb716cb.

**Comment by tarrant** [comment]
Hmm, I had zero issues with this when I wrote https://github.com/rust-lang/rust.vim/commit/eb716cba6b8a6b8e156cdee0018c92a29a5a78d8 on my other computer. Just upgraded on this computer and I'm seeing the same issue. Hopefully I can narrow down why this is happening shortly. Apologies for introducing this bug!

**Comment by tarrant** [comment]
It looks like the `augroup` stuff just really struggles being in a multi-line command. It could be wrapped in a function and then the function could be deleted, or it could just be left alone and not included in the post-buffer cleanup. Which should be fine since the autocommands are defined on a `*.rs` file only basis.

Thoughts opinions?

**Comment by tarrant** [comment]
So the more I've thought about it the more it seems that removing the `augroup` stuff from the `b:undo_ftplugin` is the right way to go. These are already correctly cleaned up via other methods and there doesn't seem to be a clean way (that I can find) to deal with them here.

I'm hesitant to fix this without some feedback since I'm the one that broke it in the first place :( but if no one responds I'll submit a patch anyways.

**Comment by tmccombs** [comment]
This happens to me in neovim, with slightly different message:

> Error detected while processing function <SNR>7_LoadFTPlugin:
> line    2:
> E216: No such group or event: rust.vim|augroup! rust.vim

**Comment by k-mack** [comment]
I don't get these errors when I load a Rust file, but I get something similar when I run `:RustFmt`
> Error detected while processing function <SNR>10_LoadFTPlugin:
> line    2:
> E216: No such group or event: rust.vim|augroup! rust.vim

**Comment by tarrant** [comment]
I have in theory a fix on my fork at [tarrant/rust.vim](https://github.com/tarrant/rust.vim). Mind giving it a try and seeing if it resolves your issues? The more people that test it the better I'll feel about opening a PR.

**Comment by k-mack** [comment]
@tarrant your fork doesn't result in the errors I was seeing before, so that's good. BTW, I am using Vim 7.4.

**Comment by GustavoKatel** [comment]
Changing buffer to one with a rust file doesn't give error messages anymore with the fix. Thanks

**Comment by tmccombs** [comment]
The fix is working for me.

**Comment by rekka** [comment]
Seems to be fixed by 77065825650128ff48a724673e6335878d68c7fa, thanks!
